### DISCUSSION TITLE
Does WebUI support Android and iOS?

### DISCUSSION THREAD

**Discussion by sammonius** [webui-dev/webui#154] [Q&A] [answered]
Are the mobile operating systems supported?

**Comment by AlbertShown** [comment, marked as answer]
No, I don't think so. The browsers on the mobile OS work/run entirely differently. Your best option is to use the native OS [Android-WebView](https://developer.android.com/reference/android/webkit/WebView).

**Comment by ovo-Tim** [comment]
Or try tauri2.
